content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,127 @@
+includes cpu/memory requests and limits for containerized workloads.
+
+There are two versions of cgroups in Linux: cgroupv1 and cgroup v2. cgroup v2 is
+the new generation of the `cgroup` API.
+
+<!-- body -->
+
+
+## What is cgroup v2? {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+cgroup v2 is the next version of the Linux `cgroup` API. cgroup v2 provides a
+unified control system with enhanced resource management
+capabilities.
+
+cgroup v2 offers several improvements over cgroup v1, such as the following:
+
+- Single unified hierarchy design in API
+- Safer sub-tree delegation to containers
+- Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+- Enhanced resource allocation management and isolation across multiple resources
+  - Unified accounting for different types of memory allocations (network memory, kernel memory, etc)
+  - Accounting for non-immediate resource changes such as page cache write backs
+
+Some Kubernetes features exclusively use cgroup v2 for enhanced resource
+management and isolation. For example, the
+[MemoryQoS](/blog/2021/11/26/qos-memory-resources/) feature improves memory QoS
+and relies on cgroup v2 primitives.
+
+
+## Using cgroup v2 {#using-cgroupv2}
+
+The recommended way to use cgroup v2 is to use a Linux distribution that
+enables and uses cgroup v2 by default.
+
+To check if your distribution uses cgroup v2, refer to [Identify cgroup version on Linux nodes](#check-cgroup-version).
+
+### Requirements
+
+cgroup v2 has the following requirements:
+
+* OS distribution enables cgroup v2
+* Linux Kernel version is 5.8 or later
+* Container runtime supports cgroup v2. For example:
+  * [containerd](https://containerd.io/) v1.4 and later
+  * [cri-o](https://cri-o.io/) v1.20 and later
+* The kubelet and the container runtime are configured to use the [systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver)
+
+### Linux Distribution cgroup v2 support
+
+For a list of Linux distributions that use cgroup v2, refer to the [cgroup v2 documentation](https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md)
+
+<!-- the list should be kept in sync with https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md -->
+* Container Optimized OS (since M97)
+* Ubuntu (since 21.10, 22.04+ recommended)
+* Debian GNU/Linux (since Debian 11 buster)
+* Fedora (since 31)
+* Arch Linux (since April 2021)
+* RHEL and RHEL-like distributions (since 9)
+
+To check if your distribution is using cgroup v2, refer to your distribution's
+documentation or follow the instructions in [Identify the cgroup version on Linux nodes](#check-cgroup-version).
+
+You can also enable cgroup v2 manually on your Linux distribution by modifying
+the kernel boot arguments in the GRUB command line, and setting
+`systemd.unified_cgroup_hierarchy=1`. However, the recommended approach is to use a
```

Comment:
It is unclear to me where to set the `systemd.xxx` argument.

Reply:
Updated this with some more details.